independent-publisher-connectors/The Lord of the Rings/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1635 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "The Lord of the Rings",
+    "description": "This is the one API to rule them all. No really, it serves your needs regarding data about The Lord of the Rings, the epic books by J. R. R. Tolkien and the official movie adaptions by Peter Jackson.",
+    "version": "1.0",
+    "contact": {
+      "name": "Troy Taylor",
+      "url": "https://www.hitachisolutions.com",
+      "email": "ttaylor@hitachisolutions.com"
+    }
+  },
+  "host": "the-one-api.dev",
+  "basePath": "/v2",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/book": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",

```

Comment:
Please do not use default responses. Add description for responses. Everything else seems to be in order. Looking forward to certifying your connector.